guestbook/static/js/guestbookapp/view/templates/GuestbookView.html

```diff
@@ -0,0 +1,75 @@
+<div>
+    <div>
+        <table>
+            <tr><td>Guestbook name: <td>
+                <td><input data-dojo-attach-point="guestbook_nameNode" id="guestbook"
```

Comment:
- rename lại toàn bộ `data-dojo-attach-point` , 
  - đặt tên theo variable name convention của JS
  - nếu là widget thì không có postfix `Node`
  - nếu là DOM html thì thêm postfix `Node`